### PR TITLE
Assignment submission

### PR DESCRIPTION
**Pull request by vaibhav-trq**

### CODE REVIEW

**Review by hellovai — APPROVED** [review]
This looks great! Nice job.

**Review by jackarendt — APPROVED** [review]
This looks great! Nice job.

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by hellovai — APPROVED** [review]
This looks great! Nice job.

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by hellovai — APPROVED** [review]
This looks great! Nice job.

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by hellovai — APPROVED** [review]
This looks great! Nice job.

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp [hunk 1]
```diff
@@ -3,5 +3,20 @@
 using namespace std;
 
 int main() {
-  cout<<"Hello "; cout <<"World!"; cout << endl;
+  cout << "Hello World!" << endl;
```
Comment (hellovai) [inline fix-code-0/main.cpp:6]: Add some spacing between here!

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp [hunk 2]
```diff
@@ -3,5 +3,20 @@
 using namespace std;
 
 int main() {
```
Comment (hellovai) [inline fix-code-0/main.cpp:5]: fads

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (hellovai) [inline fix-code-0/main.cpp:6]: fsadf

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (hellovai) [inline fix-code-0/main.cpp:6]: fsadf

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (hellovai) [inline fix-code-0/main.cpp:6]: fsadf

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp [hunk 3]
```diff
@@ -3,5 +3,20 @@
...
-  cout<<"Hello "; cout <<"World!"; cout << endl;
+  cout << "Hello World!" << endl;
 }
```
Comment (hellovai) [inline fix-code-0/main.cpp:7]: afsdf

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp (hunk 3 above)
Comment (hellovai) [inline fix-code-0/main.cpp:7]: afsdf

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:6]: Add some spacing between here!

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:6]: Add some spacing between here!

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:6]: Add some spacing between here!

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:6]: Fixed.

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: invent-code-0/main.cpp [hunk 4]
```diff
@@ -4,4 +4,5 @@ using namespace std;
 
 int main() {
```
Comment (hellovai) [inline invent-code-0/main.cpp:5]: Does this comment make it though?

**Review by hellovai — CHANGES_REQUESTED** [review]
Please address all of the comments.

File: fix-code-0/main.cpp (hunk 1 above)
Comment (hellovai) [reply fix-code-0/main.cpp:6]: awesome fix!

File: fix-code-0/main.cpp (hunk 3 above)
Comment (hellovai) [reply fix-code-0/main.cpp:7]: This isn't very useful...

File: invent-code-0/main.cpp (hunk 4 above)
Comment (hellovai) [reply invent-code-0/main.cpp:5]: Yep! Got it on my end :)

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 2 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:5]: Fixed.

**Review by vaibhav-trq — COMMENTED** [review]
Adding new comments.

File: fix-code-0/main.cpp (hunk 2 above)
Comment (vaibhav-trq) [reply fix-code-0/main.cpp:5]: Fixed.

**Pull request closed without merging**